src/dotnet/ToolPackage/IToolPackageInstaller.cs

```diff
@@ -14,5 +14,10 @@ internal interface IToolPackageInstaller
             VersionRange versionRange = null,
             string targetFramework = null,
             string verbosity = null);
+
+        IToolPackage InstallPackageToExternalManagedLocation(PackageLocation packageLocation, PackageId packageId,
```

Comment:
Nit: could we place the two parameters on their own lines like the remaining parameters (can change the `InstallPackage` above).  I don't like having multiple on one line and then only single parameters on remaining lines.

Nit: Could this simply be named `InstallExternalPackage`?  Just a suggestion, though.